src/test/java/org/jabref/logic/importer/fetcher/ArXivTest.java

```diff
@@ -41,7 +41,7 @@ public void setUp() {
         sliceTheoremPaper.setField("title", "Slice theorem for Fréchet group actions and covariant symplectic field theory");
         sliceTheoremPaper.setField("date", "2014-05-09");
         sliceTheoremPaper.setField("abstract", "A general slice theorem for the action of a Fr\\'echet Lie group on a Fr\\'echet manifolds is established. The Nash-Moser theorem provides the fundamental tool to generalize the result of Palais to this infinite-dimensional setting. The presented slice theorem is illustrated by its application to gauge theories: the action of the gauge transformation group admits smooth slices at every point and thus the gauge orbit space is stratified by Fr\\'echet manifolds. Furthermore, a covariant and symplectic formulation of classical field theory is proposed and extensively discussed. At the root of this novel framework is the incorporation of field degrees of freedom F and spacetime M into the product manifold F * M. The induced bigrading of differential forms is used in order to carry over the usual symplectic theory to this new setting. The examples of the Klein-Gordon field and general Yang-Mills theory illustrate that the presented approach conveniently handles the occurring symmetries.");
-        sliceTheoremPaper.setField("eprint", "1405.2249v1");
+        sliceTheoremPaper.setField("eprint", "http://arxiv.org/abs/1405.2249v1");
```

Comment:
This is wrong... it should just insert the number (i.e. the fetcher should be fixed not the test)